codeadmin.go

```diff
@@ -121,38 +121,39 @@ func (d *DevClient) SetServiceEffectiveUser(systemKey, name, userid string) erro
 }
 
 //UpdateService facillitates changes to the service's code
-func (d *DevClient) UpdateService(systemKey, name, code string, params []string) (error, map[string]interface{}) {
+func (d *DevClient) UpdateService(systemKey, name, code string, params []string) (map[string]interface{}, error) {
```

Comment:
@sh3nan1gans All these func signatures are changed. Won't this cause backwards compatibility issues?

Reply:
Yea, but that's okay because peeps should be vendoring these packages anyway

Reply:
Just wanted to confirm, does this Go-SDK branch works with cb_console and clearblade(blade-runner as well) repo? If not, then even though we vendor these packages, we get restricted by not being able to change the version.

Reply:
Here's the PR for cb_console - https://bitbucket.org/clearblade/cb_console/pull-requests/438/dep-ensure

I'll update the clearblade repo as well

Reply:
I'll ask @ms1752 if he's using Go-SDK in any of his adapters. They might be affected, in case he changes the version of imported Go-SDK in the near future.

Reply:
Sure. Keep in mind that it's a very easy change to make